### PR TITLE
Add Travis configuration

### PR DESCRIPTION
**Pull request by thiemowmde**
Mostly copy-pasted from https://github.com/wmde/WikibaseDataModel/blob/master/.travis.yml, except for the notifications. Also there is nothing about coverage, as it is not relevant here.

### CODE REVIEW

**Comment by lucaswerkmeister** [comment]
Hm, I don’t see anything coverage-related in the WikibaseDataModel file – the main difference I can see is the missing notifications.

This commit itself looks good to me, but now I’m confused about the commit message…

**Comment by thiemowmde** [comment]
That was a mistake. I looked at several existing Travis configurations, and many of them contain a section about test coverage.

File: Wikibase/Tests/WikibaseStandardTest.php [hunk 1]
```diff
@@ -53,6 +53,7 @@ public function testCodeSnifferStandardFiles( $sniff, $file ) {
 		$config = new Config();
 		$config->standards = [ __DIR__ . '/..' ];
 		$config->sniffs = [ $sniff ];
+		$config->reportWidth = 140;
```
Comment (lucaswerkmeister) [inline Wikibase/Tests/WikibaseStandardTest.php:56]: Intentional addition?

Comment (thiemowmde) [reply Wikibase/Tests/WikibaseStandardTest.php:56]: Yes. Without this, Travis uses a default of 80, which changes all test outputs.

**Pull request merged**